### DISCUSSION TITLE
Looking for a simple ti.identity keychain example

### DISCUSSION THREAD

**Discussion by AceMacLoud** [appcelerator/titanium_mobile#13937] [Q&A]
Hi,

I'm looking for some simple example code on how to use the ti.identity module to store and retrieve keychain items on Android.

I don't need any of the biometric features, just save a string to the keychain and read it again later.

Initialising the keychain item and saving data to it works fine, I'm working on the reading part.

I've been using bencoding.securely in the past, which just synchronously returns the value from the keychain.

From what I've seen, ti.identity works asynchronously and I've started reworking my Code around that.

However I'd love to see some sample code to maybe save me a lot of time.

Thanks!

**Comment by m1ga** [comment]
Hi, did you check the included examples: https://github.com/tidev/titanium-identity/tree/master/Example ?
